sort/comb_sort.py

```diff
@@ -18,7 +16,7 @@ def swap(i, j):
     shrink = 1.3
     sorted = False
     while not sorted:
-        gap = int(floor(gap/shrink))
+        gap = int(gap / shrink)
```

Comment:
Thanks for the edits. 
I think you missed this line. This line could be `gap //shrink`.

Reply:
Because shrink is not a int, so gap // shrink is not a int neither.

Reply:
oh I see. Thanks for pointing out.